wiki/Accuracy/en.md

```diff
@@ -1,96 +1,125 @@
-| (Number of droplets + Number of 100s + Number of 300s) | (Number of misses + Number of droplets + Number of 100s + Number of 300s + Number of miss droplets) | "Bananas" (Spinner's fruits) do not count. |
+Term | Formulae
+:---:|:---
+**Total points of hits** | ((Number of Miss * 0) + (Number of 100(GOOD) * 0.5) + (Number of 300(GREAT) * 1)) * 300
+**Total number of hits** | Number of Miss + Number of 100 + Number of 300
 
-Note that if you use the API to calculate the accuracy, number of droplets is under **count50** and number of miss droplets is under **countkatu**
+### osu!catch
 
-Mania
----------
+In [osu!catch][osu!catch wikilink], the song accuracy is calculated as the number of non-spinner objects collected divided by the total number of non-spinner objects; fruits, large drops and small drops (droplets) all have the same value.
 
-Accuracy is calculated similarly to [Standard](/wiki/Game_Modes/osu!) in this mode.
+In other words: **Accuracy = Total numbers of fruits caught / Total number of fruits**
 
-**Accuracy = Total points of hits / (Total number of hits * 300)**
+Term | Formulae
+:---:|:---
+**Total numbers of fruits caught** | Number of droplets + Number of drops + Number of fruits
+**Total numbers of fruits**        | Number of misses + Number of miss droplets + Number of droplets + Number of drops + Number of fruits
 
-| Total points of hits | Total number of hits |
-| -------------------- | -------------------- |
-| (Number of droplets + Number of 100s + Number of 300s) | (Number of misses + Number of 50s + Number of 100s + Number of 200s + Number of 300s + Number of MAXes) |
+"Bananas" (Spinner's fruits) do not count.
 
-Note that MAX and 300 are both worth the maximum for calculating accuracy, despite a MAX being worth more in terms of score than a 300.
+Note that if using the API to calculate the accuracy, number of droplets is under **count50** and number of miss droplets is under **countkatu**
 
-Results screen
------------------
+### osu!mania
+
+In [osu!mania][osu!mania wikilink], accuracy is calculated similarly to [osu!standard][osu! wikilink] in this mode.
+
+In other words: **Accuracy = Total points of hits / (Total number of hits * 300)**
+
+Term | Formulae
+:---:|:---
+**Total points of hits** | (Number of 50s * 50 + Number of 100s * 100 + Number of 200s * 200 + Number of 300s * 300 + Number of rainbow 300s * 300)
+**Total number of hits** | (Number of misses + Number of 50s + Number of 100s + Number of 200s + Number of 300s + Number of rainbow 300s)
+
+Note that MAX (or rainbow 300) and 300 are both worth the maximum for calculating accuracy, despite a MAX being worth more in terms of score than a 300.
+
+## Results screen
 
 ### Ranking
 
-  This shows your grade, an overview of your hits, as well as your actual accuracy percentage. For more details, see [Score](/wiki/Score).
+This shows the grade, an overview of the hits, as well as the actual accuracy percentage.
```

Comment:
if you're going to remove `you` might as well be:
`This shows the grade, an overview of the hits, as well as the actual accuracy percentage of the player.`